sdk/storage/azure-storage-common/src/crypt.cpp

```diff
@@ -97,7 +96,7 @@ namespace Azure { namespace Storage {
       AlgorithmProviderInstance(AlgorithmType type)
       {
         const wchar_t* algorithmId = nullptr;
-        if (type == AlgorithmType::HmacSha256 || type == AlgorithmType::Sha256)
```

Comment:
nit: With just one such type of hashing algorithm now, we could simplify the implementation a bit and remove `AlgorithmType`, but it is fine to keep that for now, if we expect new algorithms to get added and supported later.